catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/ProjectOptionsTest.java

```diff
@@ -307,7 +310,9 @@ public void saveExternal() throws IOException {
 		onView(withId(R.id.project_options_save_external))
 				.perform(ViewActions.scrollTo())
 				.perform(click());
-
+		onToast(withText(R.string.notification_save_project_to_external_storage_pending))
+				.check(matches(isDisplayed()));
+		onView(isRoot()).perform(CustomActions.wait(3000));
```

Comment:
Please also test the Toast after the export was completed. This suggestion just shows the idea, but does not work yet, because on fast phones the pending Toast is not shown before the finished Toast. Instead the Toast "Saved project to ..." is immediately shown, therefore the test fails on fast phones. Maybe you can fix this, so that it works on slow and fast devices.
```suggestion
		onToast(withText(pendingText))
				.check(matches(isDisplayed()));
		onView(isRoot()).perform(CustomActions.wait(3000));
		onToast(withText(finishedText))
				.check(matches(isDisplayed()));
		onView(isRoot()).perform(CustomActions.wait(3000));
```